src/platform/forms-system/src/js/review/ArrayField.jsx

```diff
@@ -64,22 +64,26 @@ class ArrayField extends React.Component {
     return schema.additionalItems;
   }
 
-  scrollToTop() {
-    setTimeout(() => {
-      // Hacky; won’t work if the array field is used in two pages and one isn’t
-      //  a BasicArrayField nor if the array field is used in three pages.
-      scroller.scrollTo(
-        `topOfTable_${this.props.path[this.props.path.length - 1]}${
-          this.isLocked() ? '_locked' : ''
```

Comment:
I didn't include this `isLocked` addition since the updated `scrollToAndFocus` function is only called on removing or saving after editing, both of which are available in not-locked fields.